content/projects/understanding-loops/_index.md

```diff
@@ -65,6 +65,23 @@ Please note that one person can have multiple baskets. If you look at the data,
 
 ## Write some functionality
 
+### Important notes for Javascript solutions
+
+In Javascript it is best practice to use destructuring when passing arguments to a function. You can learn about destructuring here: [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).
+
+Please make sure you use destructuring for all your functions. 
+
+```
+getCustomerBaskets(cart, "sine@umuzi.org"); // DON'T DO THIS
+getCustomerBaskets("sine@umuzi.org", cart); // DON'T DO THIS EITHER
+
+// The problem with the above code is that if you get the order of the parameters wrong, then things will break. Your functions should work like this instead:
+
+function getCustomerBaskets({ cart, email }) {...}
+
+getCustomerBaskets({cart, email}) //DO THIS
+``` 
```

Comment:
Also, this contradicts later instructions.

```

Write a function called `get all customers`, the function should take the data array as an argument and should return a list of customer email addresses. The list must have no duplicates.
e.g. `getAllCustomers(dataStore);`
```

The example ises javascript naming conventions, not python ones. But then it doesn't use restructuring even though restructuring is required.

Ideal solution:
- the instructions should work for all languages
- the instructions should be consistent and not contradictory in any way